src/ripple/rpc/impl/Handler.cpp

```diff
@@ -104,6 +104,7 @@ Handler const handlerArray[] {
     {   "submit_multisigned",   byRef (&doSubmitMultiSigned),   Role::USER,  NEEDS_CURRENT_LEDGER  },
     {   "server_info",          byRef (&doServerInfo),          Role::USER,  NO_CONDITION     },
     {   "server_state",         byRef (&doServerState),         Role::USER,  NO_CONDITION     },
+    {   "crawl_shards",         byRef (&doCrawlShards),         Role::USER,  NO_CONDITION     },
```

Comment:
I'm not sure this should be `Role::USER`.

Reply:
Should only admins have access?

Reply:
Thinking about this more, it seems requiring an admin role is a good idea. We can always reduce the requirement in a future release.